networks/rpc/handler.go

```diff
@@ -427,6 +428,9 @@ func (h *handler) handleSubscribe(cp *callProc, msg *jsonrpcMessage) *jsonrpcMes
 func (h *handler) runMethod(ctx context.Context, msg *jsonrpcMessage, callb *callback, args []reflect.Value) *jsonrpcMessage {
 	result, err := callb.call(ctx, msg.Method, args)
 	if err != nil {
+		if UpstreamArchiveEN != "" && shouldRequestUpstream(err) {
```

Comment:
LGTM. Regarding this feature, let me ask three usage questions.

1. The single URL may be extended to the list of URL. Was there an issue regarding this?
2. The URL (list or single) seems not modifiable at runtime. If the initially set node becomes unreachable, the routing node should be restarted to change that value. It seems to degrade the usage.
3. Any request can be relayed as well as the state lack. Is this an obvious limitation by the consideration of big load to the upstream node?

Reply:
Got it thanks.

Reply:
@sjnam can you add todo about those suggestions? Also, not mentioned here, additional suggestion is to make a new rpcErrorResponse struct for it.